src/commands/collection/dump.ts

```diff
@@ -26,6 +26,15 @@ export default class CollectionDump extends Kommand {
   ]
 
   async run() {
+    try {
```

Comment:
(nitpicking / open to debate) there are so many of these everywhere, and it's so generic, that I think that this `run` command should be put in the `Kommand` parent class instead. 

If you do so, I would reverse the names because the safe runner is this one, not the one invoked by it:
* the Kommand class should have a `runSafe`method with almost the same code as the one I'm highlighting in this comment (only change is it would invoke "run" instead of "runSafe" obviously)
* the Kommand class should have  a `run` method throwing a "Not implemented" error when invoked
* other classes should overload the "run" command as they see fit
* to use these functions, we can chose to explicitly call "run" or "runSafe" (I understand that only the latter shall be invoked but, you know...)

Reply:
The `run` command is called internally by oclif so I will put the `try.. catch` code inside the `Kommand` class and call the class `runUnsafe` method so it will avoid code duplication.

But I will apply these change in a separate PR So I can correct all the new method at once :sweat_smile: